sources/packages/api/src/testHelpers/controllers/auth-test/auth-test.controller.ts

```diff
@@ -42,7 +45,11 @@ export class AuthTestController {
    */
```

Comment:
I think, the method does not have `* @param userToken`

Reply:
The parameter userToken is there but missing a comment. Is it about the missing comment?

Reply:
Comment added.